src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -102,6 +103,27 @@ export class SkyRepeaterItemComponent implements OnDestroy, OnInit {
   @Output()
   public isSelectedChange = new EventEmitter<boolean>();
 
+  public set childFocusIndex(value: number) {
+    const focusableChildren = this.adapterService.getFocusableChildren(
```

Comment:
You're making the same call three different times. Could you simplify the adapter service method to the following?
```
this.adapterService.getFocusableChildren(this.itemRef)?
```

Reply:
Done.